src/util.js

```diff
@@ -17,3 +35,11 @@ const checkForErrors = (response) => {
   }
 } 
 
+export const cleanData = (movieObj) => {
```

Comment:
This is an elegant way to do this, and will help us with our CSS styling so everything doesn't go bananas when some info is missing.